### ISSUE TITLE
Option to remove LayoutPlugin

### ISSUE THREAD

**Issue by paulopicasso** [ookami-kb/storybook_flutter#124]
Maybe I'm missing something but is there an option to remove the layout plugin? We're developing a purely mobile app and don't even support landscape mode, so the button is useless to us, but accidentally tapping it twice really screws up the layout.

**Comment by ookami-kb** [comment]
Hey, yep, I don't think it's possible at the moment. I will take a look into it.

**Comment by paulopicasso** [comment]
Thanks!

**Comment by bkkoluch** [comment]
I agree, imo all plugins should be optional - the same for `LoggingPlugin` in `0.14.1`
